docs/bonsai-ui/checkout/index.mdx

```diff
@@ -199,6 +199,12 @@ type Config = {
   originUrl?: string;
   /* ID of the order provided to Bonsai by the party who placed the order - should be unique per order if passed. */
   externalOrderId?: string;
+  /*
+   * An array of allowed countries for checkout. If provided, the checkout will be restricted to
+   * these countries. Otherwise defaults to our supported countries.
+   * Use ISO 3166-1 alpha-2 country codes (e.g., 'US' for United States, 'CA' for Canada).
```

Comment:
```suggestion
   * An array of allowed countries for checkout. If provided, the checkout shipping form will be restricted to
   * these countries. Otherwise defaults to our supported countries.
   * Use ISO 3166-1 alpha-2 country codes (e.g., 'US' for United States, 'CA' for Canada).
```